entry_check.php

```diff
@@ -0,0 +1,31 @@
+<?php
+require_once('functions.php');
+session_start();
+
+$db = getDB();
+$science_name = $_POST['science_name'];
```

Comment:
this will cause an error if the key doesnt exist in post. you should do the empty checks on the `$_POST` variable rather than assigning to variables